CMakeLists.txt

```diff
@@ -84,21 +84,21 @@ install(TARGETS astra_camera_node
 )
 
 install(TARGETS astra_wrapper astra_driver_lib
-  ARCHIVE DESTINATION ${CMAKE_INSTALL_PREFIX}/lib
-  LIBRARY DESTINATION ${CMAKE_INSTALL_PREFIX}/lib
-  RUNTIME DESTINATION ${CMAKE_INSTALL_PREFIX}/bin
+  ARCHIVE DESTINATION lib
+  LIBRARY DESTINATION lib
+  RUNTIME DESTINATION bin
 )
 
 install(FILES ${CMAKE_BINARY_DIR}/openni2/Redist/libOpenNI2Orbbec.so
-  DESTINATION ${CMAKE_INSTALL_PREFIX}/lib
+  DESTINATION lib
 )
 
 install(DIRECTORY ${CMAKE_BINARY_DIR}/openni2/Redist/OpenNI2
-  DESTINATION ${CMAKE_INSTALL_PREFIX}/lib
+  DESTINATION lib
 )
 
 install(DIRECTORY include/${PROJECT_NAME}
-  DESTINATION ${CMAKE_INSTALL_PREFIX}/include/
+  DESTINATION include/
```

Comment:
I would suggest to remove the trailing slash since it has no special meaning here and might just confuse readers.